### PR TITLE
Add support for setting data dir by env var AFFECTIVA_VISION_DATA_DIR

### PR DESCRIPTION
**Pull request by tmoopenn**
- same print and error statements as in cpp-sdk-samples regarding setting of data directory

### CODE REVIEW

File: python-sdk-samples/affvisionpy-sample.py [hunk 1]
```diff
@@ -107,6 +108,11 @@ def get_command_line_parameters(args):
     else:
         input_file = int(args.camera)
     data = args.data
+    if not data:
+        data = os.environ.get(DATA_DIR_ENV_VAR)
+        if data == None:
+            raise ValueError("Data directory not specified via command line or env var")
```
Comment (fkberthold) [inline python-sdk-samples/affvisionpy-sample.py]: Would it be useful to note what the name of the environment variable is in the error?

File: python-sdk-samples/affvisionpy-sample.py [hunk 2]
```diff
@@ -631,7 +637,7 @@ def parse_command_line():
     parser = argparse.ArgumentParser(description="Sample code for demoing affvisionpy module on webcam or a saved video file.\n \
         By default, the program will run with the camera parameter displaying frames of size 1280 x 720.\n \
         A CSV file will also be written by default with the filename 'default.csv'. ")
-    parser.add_argument("-d", "--data", dest="data", required=True, help="path to directory containing the models")
+    parser.add_argument("-d", "--data", dest="data", required=False, help="path to directory containing the models")
```
Comment (adennie) [inline python-sdk-samples/affvisionpy-sample.py]: in this help text, indicate the alternative of using the env var, like we do in the cpp-sdk-samples:
https://github.com/Affectiva/cpp-sdk-samples/blob/auto-sdk-1.2/vision/frame-detector-video-demo/VideoDemo.cpp#L136

File: python-sdk-samples/affvisionpy-sample.py (hunk 1 above)
Comment (tmoopenn) [reply python-sdk-samples/affvisionpy-sample.py]: Yes, I think that's a good idea. I'll modify python and cpp samples to reflect the change

File: python-sdk-samples/affvisionpy-sample.py (hunk 2 above)
Comment (tmoopenn) [reply python-sdk-samples/affvisionpy-sample.py]: Yes, good catch. I realized this as well this morning. Making the edit and will add in README as well.

File: python-sdk-samples/affvisionpy-sample.py (hunk 1 above)
Comment (tmoopenn) [reply python-sdk-samples/affvisionpy-sample.py]: @adennie @umangmehta12 @ahmedalmoraly Do you have any objections to this change?

**Comment by tmoopenn** [comment]
- Changed error messages when data directory is not specified or incorrect path from raising value error that gives a backtrace to printing an error message and displaying help message. This mirrors more closely the behavior of the cpp-samples.
- update readme 
- print name of environment variable in error message.

File: README.md [hunk 3]
```diff
@@ -16,31 +16,31 @@ This script demonstrates how to use Affectiva's affvisionpy module to process fr
         python3 affvisionpy-sample.py [-h] -d DATA [-i VIDEO] [-n NUM_FACES] [-c [CAMERA]] [-o OUTPUT] [-f FILE] [-r WIDTH HEIGHT]
 
         required arguments:
-            -d DATA, --data DATA  path to directory containing the models
-        
+            -d DATA, --data DATA  path to directory containing the models. Alternatively specify path via the environment variable AFFECTIVA_VISION_DATA_DIR=/path/to/data
+
```
Comment (umangmehta12) [inline README.md]: Since its the auto-sdk, it would be `/path/to/data/vision`.

File: python-sdk-samples/affvisionpy-sample.py (hunk 1 above)
Comment (ahmedalmoraly) [reply python-sdk-samples/affvisionpy-sample.py]: i think it's a good idea 👍

Comment (tmoopenn) [reply python-sdk-samples/affvisionpy-sample.py]: @umangmehta12 yes that's correct

**Comment by umangmehta12** [comment]
Looks good to me apart from the doc string change

File: python-sdk-samples/affvisionpy-sample.py [hunk 4]
```diff
@@ -635,9 +640,9 @@ def parse_command_line():
     args: argparse object of the command line
     """
     parser = argparse.ArgumentParser(description="Sample code for demoing affvisionpy module on webcam or a saved video file.\n \
-        By default, the program will run with the camera parameter displaying frames of size 1280 x 720.\n \
-        A CSV file will also be written by default with the filename 'default.csv'. ")
-    parser.add_argument("-d", "--data", dest="data", required=False, help="path to directory containing the models")
+        By default, the program will run with the camera parameter displaying frames of size 1280 x 720.\n")
+    parser.add_argument("-d", "--data", dest="data", required=False, help="path to directory containing the models. \
+                        Alternatively, specify the path via the environment variable " + DATA_DIR_ENV_VAR + "=/path/to/data")
```
Comment (umangmehta12) [inline python-sdk-samples/affvisionpy-sample.py]: @tmoopenn this needs to be "/path/to/data/vision" as per the PR review.

Comment (tmoopenn) [reply python-sdk-samples/affvisionpy-sample.py]: thanks for the check, I'll change that.

**Pull request merged**